gravityflow/gravityflow
# Add inbox methods to the Gravity_Flow_API

Added the `Gravity_Flow_API::get_inbox_entries()` and `Gravity_Flow_API::get_inbox_entries_count()` methods along with helpers for getting the filter key, search criteria, form IDs, paging, and sorting arguments.

Unit tests for all the new methods are included.

**Testing instructions**

- Access the inbox page with master take note of which entries are listed. 
- Update to this branch and confirm the same entries are listed.
- Check the `gravityflow_inbox_search_criteria`, `gravityflow_form_ids_inbox`, `gravityflow_inbox_paging`, and `gravityflow_inbox_sorting` filters still function.
